### ISSUE TITLE
IceWeasel error "Failed to create DBus proxy"

### ISSUE THREAD

**Issue by grexe** [haikuports/haikuports#12622]
I see a lot of these warnings in the command line output:
```
Failed to create DBus proxy[Parent 490, Main Thread] WARNING: Failed to create DBus proxy for org.a11y.Bus: Cannot spawn a message bus when AT_SECURE is set
: 'glib warning', file /sources/firefox-139.0.1/toolkit/xre/nsSigHandlers.cpp:203
```
using IceWeasel 139.0.1 on latest Haiku nightly hrev58979
